internal/common/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.14
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
 	github.com/Microsoft/go-winio v0.4.14 // indirect
+	github.com/aws/aws-sdk-go v1.31.9
```

Comment:
This will cost us because we enforce aws dependency everywhere (and we do have azure there as well). Probably we should split "internal/common" per vendor if any vendor specific code is here.

Do you mind filing an issue and help us to split common to isolate any vendor specific code?

Reply:
By "cost", I presume you meant the resulting collector-contrib binary size will increase. I can create an issue and possibly a separate PR for that.

Is it possible that we get this merged first to unblock my other PR #691 ?